modules/Microsoft.Web/staticSites/.bicep/nested_roleAssignments.bicep

```diff
@@ -4,9 +4,9 @@ param roleDefinitionIdOrName string
 param resourceId string
 
 var builtInRoleNames = {
-  'Owner': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '8e3af657-a8ff-443c-a75c-2fe8c4bcb635')
-  'Contributor': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
-  'Reader': subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'acdd72a7-3385-48ef-bd42-f606fba81ae7')
+  Owner: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '8e3af657-a8ff-443c-a75c-2fe8c4bcb635')
```

Comment:
Why this change?

Reply:
Hey @eriqua, this is aligned with Bicep best practice and avoids this warning message:

```
Warning prefer-unquoted-property-names: Property names that are valid identifiers should be declared without quotation marks and accessed using dot notation. [https://aka.ms/bicep/linter/prefer-unquoted-property-names]
```

Reply:
Please undo, this is a new warning message with is disabled with PR #1632.

Reply:
Why are we not complying with Bicep best practices here @MrMCake? When just importing some of the modules in a project, the current `bicepconfig.json` is probably not going to be copied or has it's own bicepconfig.json. Also how does this behave with CARML modules consumed from a ACR?
I think we should comply with the Bicep linter wherever possible, I don't see any benefit in deviating, but downsides for all CARML customers that will receive warnings in their projects.

Reply:
The rule was just introduced, which means all modules now produce this warning. 
No matter whether one thinks the rule is good or not (in this case I, for example, prefer that all roles use the same syntax), I'd much rater prefer consistency across the modules and change them all at ones instead of one module at a time.

Reply:
I'd suggest a new issue as also #1632 does not involve the main source of the warnings: the RBAC roles ;)